### ISSUE TITLE
Cleanup multiline strings

### ISSUE THREAD

**Issue by bakkdoor** [bakkdoor/fancy#51]
In case of multiline strings, especially when used for documentation strings in class or method definitions, we should clean up the string so that for example any preceeding whitespace (due to indentation in the sourcefile) gets removed. If there's indentation for documentation purposes that differs from the overall "starting column" of the other lines, preserve the additional indentation and so on...
Basically, make docstrings look good in fdoc generated api documentation.

**Comment by vic** [comment]
Done, the FDoc Formatter does this.
